app/Http/Controllers/SanatizerController.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Support\Facades\File;
+
+class SanatizerController extends Controller
+{
+
+    public function __construct()
+    {
+        $this->rawHTMLFolderPath = storage_path('nonSanatizedSuras');
+        $this->cleanedSurasPath = storage_path('SanatizedSuras');
+        
+    }
+
+    public function sanatize()
+    {
+        $rawHTMLFiles = scandir($this->rawHTMLFolderPath);
+        foreach ($rawHTMLFiles as $rawHTMLFile) {  
+            if (($rawHTMLFile != '.')&&($rawHTMLFile != '..')) {
+                $suraStringToClean = self::getSuraString($rawHTMLFile); 
+                $readyToSaveSura = self::cleanSuraString($suraStringToClean);
+                
+
+                $fileNameToSave = substr($rawHTMLFile, 0, strpos($rawHTMLFile, "."));
+                $fileNameToSave = $fileNameToSave.$readyToSaveSura["suraName"];
```

Comment:
I really like the way you used the name (which represents the priority of the `sura`) in the html files :ok_hand: